### PR TITLE
Responsive home page

### LINKED ISSUE #160: Responsive Layout: Landing Page
[Project/Issue Tracker](https://github.com/Code-4-Community/lucys-love-bus-frontend/projects/1)

### Objective

As a user I would like to comfortably view the landing page on my phone. See the Figma for an exact specification of how the UI should look.
 
### Tech Spec
- Augment the CSS such that the component fits on tablet and mobile, include screenshots for verification
- Website must match Figma specifications.

Please look at #161 (PR #175) if you need starting ideas.

### PR DESCRIPTION
**Pull request by varunthakkar1**
## Issue

Closes #160 

## Changes

Adds responsiveness to the home page. Also changes some styling for the responsive single event view.

## Screenshots

image
image
image

## Verification Steps

Verify that the homepage and single event view is cleanly responsive. Check all mobile devices for any wonky styling.

### CODE REVIEW

File: src/containers/home/index.tsx [hunk 1]
```diff
@@ -104,13 +123,23 @@ const Home: React.FC<HomeContainerProps> = ({ events, announcements }) => {
   }, []);
 
   const LandingCard = styled(Card)`
-    position: relative;
-    top: 30%;
+    position: absolute;
+    top: 40%;
```
Comment (rymaju) [inline src/containers/home/index.tsx:127]: Percentages are scary, but I'm assuming you tested this on very large screen sizes too right 👀

File: src/containers/home/index.tsx [hunk 2]
```diff
@@ -28,24 +28,42 @@ import {
...
+const LandingImage = styled.div`
   width: 100%;
   height: calc(95vh - 110px);
   object-fit: cover;
   background-image: url('${HOME_IMAGE}');
   background-position: center; /* Center the image */
   background-repeat: no-repeat; /* Do not repeat the image */
   background-size: cover; /* Resize the background image to cover the entire container */
+
+  @media screen and (max-width: 900px) {
```
Comment (rymaju) [inline src/containers/home/index.tsx:45]: ⛏️ 900px should be extracted as a constant so that we can change breakpoints easily later

**Review by rymaju — APPROVED** [review]
Overall nothing too objectionable, if it looks right on mobile then it looks good to me.

**Review by ja0911 — APPROVED** [review]
Nice changes- the demo last week looked good.

**Pull request merged**